tests/main/interfaces-cups-control/task.yaml

```diff
@@ -15,7 +15,8 @@ details: |
 
 # Default cups/cups-pdf configuration on these distributions isn't
 # working yet without further tweaks.
-systems: [-ubuntu-core-*, -opensuse-*, -fedora-*, -arch-*, -amazon-*, -centos-*]
+# TODO: enable on debian-sid once https://launchpad.net/bugs/1006853 is fixed
```

Comment:
This link (https://launchpad.net/bugs/1006853) leads me to " jquery.tipsy is outdated " - this is not

Reply:
@mvo5 sorry, the issue was wrong, this is the one Till gave me https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1006853